UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Services/FileShareService.cs

```diff
@@ -107,5 +108,21 @@ public BatchStatusResponse GetBatchStatus(string batchId, string homeDirectoryPa
             }
             return batchStatusResponse;
         }
+
+        public bool CleanUp(List<string> batchId,  string homeDirectoryPath)
+        {
+            string folderName = fileShareServiceConfiguration.Value.FolderDirectoryName;
+            bool deleteFlag = false;
+            foreach (var item in batchId)
+            {
+                string uploadBlockFolderPath = Path.Combine(homeDirectoryPath, folderName, item);
```

Comment:
Kindly change variable name to exchangeSetZipFolderPath

Reply:
Did changes accordingly. Thanks!